src/stackwalk.c

```diff
@@ -240,18 +240,17 @@ JL_DLLEXPORT jl_value_t *jl_backtrace_from_here(int returnsp, int skip)
     return bt;
 }
 
+// note: btout and bt2out must be GC roots
```

Comment:
I'm a little surprised the static analyzer is ok with this; I believe these should be marked with `JL_REQUIRE_ROOTED_SLOT`? I can add that tweak in #33277 where I'm touching the same code anyway.